runhouse/resources/serverless/aws_lambda_function.py

```diff
@@ -709,9 +709,14 @@ def config_for_rns(self):
         )
         return config
 
+    @property
+    def handler_path(self):
+        return self.local_path_to_code[0]
+
 
 def aws_lambda_function(
     fn: Callable = None,
+    fn_pointers: Optional[Tuple] = None,
```

Comment:
We should be able to extract these ourselves instead of having the user provide them (assuming we are given the `fn`)

Reply:
Yeah, makes sense. Removed.